### ISSUE TITLE
Icons missing for system settings in Ubuntu

### ISSUE THREAD

**Issue by dirtydancing** [numixproject/numix-icon-theme#442]
Hi, in the course of the discussion in https://github.com/numixproject/numix-icon-theme/issues/438, the issue came up that apparently there are several system settings icons missing in Numix base.

This is a screenshot of daily Ubuntu 15.04 using Numix base (same in current Ubuntu 14.10):

![1504daily_numix](https://cloud.githubusercontent.com/assets/7164227/7049869/66e9526c-de1d-11e4-8787-8a5efb7b0278.png)

Not sure (yet) what the relevant icon names are, and whether this issue can be addressed without conflicts with the system settings icons in Numix-Circle (?). This is to open a separate issue report.

**Comment by wa4557** [comment]
Also related to #160 is inheriting Humanity an alternative?

**Comment by dirtydancing** [comment]
Yes, it should be. This issue relates to Ubuntu, and Ubuntu ships with Humanity.

One thing must be considered, though: The **differing inheritance lines**:

Currently in Ubuntu 14.10, Humanity inherits `gnome,hicolor` (which is in line with Numix base inheriting `gnome,hicolor`). But in Ubuntu 15.04, Humanity inherits `Adwaita,hicolor`, i.e. **Adwaita** instead of gnome. And because I figure that the proposed inheritance line of Numix would be `Humanity,gnome,hicolor`, there would be a deviation from that inheritance line in Humanity. This might not be a problem at all (after all, Adwaita is the successor of gnome), I just wanted to point that out.

Adwaita icon theme in current daily Ubuntu 15.04 does not have a specific inheritance line, and gnome icon theme does not either.

One thought that springs to mind with this, is that Numix could change its inheritance line to `Humanity,Adwaita,hicolor` as well, but that does not seem to be a good idea, because e.g. Ubuntu 14.10 does not ship with Adwaita icon theme (but Ubuntu 15.04 still also ships with gnome icon theme).

As an additional "bonus", inheriting from Humanity would also ensure that there are fallback icons for the Ubuntu notification icons, just in case. For Ubuntu 15.04, the notification icons have been moved from gnome to Humanity, cf. https://github.com/numixproject/numix-icon-theme/pull/298#issuecomment-83983789.

What do you make of this?

**Comment by wa4557** [comment]
What happens if you are on KDE and Humanity is not present?

**Comment by dirtydancing** [comment]
Humanity should simply be skipped if it is not present.

But this points to a problem with inheritances other than `gnome,hicolor`: Numix is not distro-specific, and depending on the distribution, this might lead to different results, also because there might be different versions of a "non-fallback" icon theme like Humanity. And as we have seen above, the inheritances of such other icon themes might change, and that might cause more bad than good. As far as I can tell, I would probably not recommend to add Humanity to the Numix/Numix-Light inheritance line.

**Comment by wa4557** [comment]
Me neither, I suppose this is then a won't fix?

**Comment by dirtydancing** [comment]
I guess so. I am not sure if Numix base is supposed to cover all these icons (complete in Numix-Circle).

**Comment by dirtydancing** [comment]
Closing this issue because these icons are located consistently in Numix-Circle.

One way to manually and individually address this issue would indeed be to modify the inheritance line to include `Humanity`. This will fill up the missing icons in the Ubuntu 15.04 (current daily) system settings: I have checked that with the inheritance line: `Inherits=Humanity,gnome,hicolor`.

Of course, even that would not fix the Bluetooth icon, and thus https://github.com/numixproject/numix-icon-theme/issues/438 is still valid.
